### PR TITLE
CLOUDP-70041: mongocli atlas maintenanceWindow(s) clear|rm

### PR DESCRIPTION
**Pull request by andreaangiolillo**
## Proposed changes

_Jira ticket:_ CLOUDP-70041

## Checklist

- [x] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added any necessary documentation (if appropriate)
- [x] I have run `make fmt` and formatted my code

## Further comments

```
./bin/mongocli atlas maintenanceWindows clear
? Are you sure you want to delete: maintenance window Yes
Maintenance window removed.
```

Screenshot 2020-09-08 at 17 30 26

### CODE REVIEW

**Comment by andreaangiolillo** [comment]
waiting for https://github.com/mongodb/mongocli/pull/422

File: internal/cli/atlas/maintenance/clear.go [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+var clearTemplate = "Maintenance window removed.\n"
+
+func (opts *ClearOpts) Run() error {
+	err := opts.store.ClearMaintenanceWindow(opts.ConfigProjectID())
+	if err != nil {
+		return err
+	}
+
+	return opts.Print(clearTemplate)
```
Comment (gssbzn) [inline internal/cli/atlas/maintenance/clear.go:46]:
```suggestion
	return opts.Print(nil)
```

File: internal/cli/atlas/maintenance/clear.go [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+}
+
+// mongocli atlas maintenanceWindow(s) clear [--projectId projectId]
+func ClearBuilder() *cobra.Command {
+	opts := &ClearOpts{}
+	cmd := &cobra.Command{
+		Use:     "clear",
+		Short:   clearMaintenanceWindow,
+		Aliases: []string{"rm", "delete"},
+		PreRunE: func(cmd *cobra.Command, args []string) error {
```
Comment (gssbzn) [inline internal/cli/atlas/maintenance/clear.go:56]: [q] should we confirm? I don't have a strong opinion right now but wanted to throw it out

**Review by gssbzn — CHANGES_REQUESTED** [review]
One small nit an a non blocking question

File: internal/cli/atlas/maintenance/clear.go (hunk 2 above)
Comment (andreaangiolillo) [reply internal/cli/atlas/maintenance/clear.go:56]: yes I think we should, I am going to change it

**Review by gssbzn — APPROVED** [review]
LGTM

**Pull request merged**